script.js

```diff
@@ -0,0 +1,78 @@
+
+const omdbAPIkey = "5af38ba4";
+var test = document.getElementById('root');
+
+function apicall() {
+    //placeholder movie + key
+    fetch('https://ghibliapi.herokuapp.com/films')
+    .then(response => {
+        return response.json();
+    })
+    //returns movie data as array
+    .then(data => {
+        console.log(data);
+    });
+}
+apicall();
+
+
+
+//example function to call OMDB api, TODO: will take specific parameters once they're built and implemented
+function buildOMDBApiCall() {
+    //placeholder movie + key
+    fetch('https://www.omdbapi.com/?t=Majo no takkyūbin&apikey=5af38ba4')
+    .then(response => {
+        return response.json();
+    })
+    //returns movie data as array
+    .then(data => {
+        console.log(data);
+    });
+}
+buildOMDBApiCall();
+
+
+
+var container = document.createElement('div');
+container.setAttribute('class', 'container');
+
+test.appendChild(container);
+
+var request = new XMLHttpRequest();
```

Comment:
Any reason to use XMLHttpRequest() here instead of fetch()?

Reply:
not in particular, fetch is fine